### PR TITLE
Move gcsweb under /apps

### PR DESCRIPTION
**Pull request by jimdaga**
## Resolves:
* https://github.com/kubernetes/k8s.io/issues/2150 (continuation) 

## Changes:
* Move `gcsweb.k8s.io` directory under the new `/apps` path. 
* Update metadata with any missing details.
* Update README instructions 

## Notes:
* Checked https://cs.k8s.io/ and didn't find any references to these files, so no external dependencies require updates.

### CODE REVIEW

File: apps/gcsweb.k8s.io/certificate.yaml [hunk 1]
```diff
@@ -2,6 +2,9 @@ apiVersion: networking.gke.io/v1
 kind: ManagedCertificate
 metadata:
   name: gcsweb
+  namespace: gcsweb
```
Comment (jimdaga) [inline apps/gcsweb.k8s.io/certificate.yaml:5]: Still researching if this is actually in the `gcsweb` namespace, or if it lives in `default` for some reason. I may need to work on getting more access into the aaa cluster to see this.

Comment (ameukam) [reply apps/gcsweb.k8s.io/certificate.yaml:5]: @jimdaga I checked : 

```shell
ameukam@cloudshell:~ (kubernetes-public)$ kubectl -n gcsweb get managedcertificate
NAME     AGE   STATUS
gcsweb   40d   Active
```

Comment (jimdaga) [reply apps/gcsweb.k8s.io/certificate.yaml:5]: Great, thanks for checking that @ameukam!

**Review by spiffxp — APPROVED** [review]
/ok-to-test
/approve
/lgtm

**Comment by ameukam** [comment]
FYI @thockin

**Pull request merged**